Sources/SotoCore/HTTP/S3StreamWriter.swift

```diff
@@ -0,0 +1,140 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2017-2020 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+import SotoSignerV4
+
+class S3StreamWriter: StreamWriterProtocol {
+    /// Working buffer size
+    static let bufferSize: Int = 64 * 1024
```

Comment:
Out of curiosity why have we picked these numbers? If it's an S3 thing we should probably link to the docs

Reply:
This is the recommended size from AWS. See https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html

Reply:
I think we should add that as a comment